src/serializer/ResidualHeapVisitor.js

```diff
@@ -729,11 +729,12 @@ export class ResidualHeapVisitor {
         instance: funcInstance,
       });
       this.visitGenerator(generator);
-      this._withScope(generator, visitPropertiesAndBindings);
+      // All modified properties and bindings should be accessible
```

Comment:
The comment sounds reasonable. Should the `visitGenerator` call one line earlier also get the same treatment? If not, why not?

Reply:
Maybe not, "generator" above represents any time sensitive generator entries captured in the additional function so it should be visited from that generator(or its sub-generators).
While for modified properties and bindings that are not captured in generator, they should be in additional function's scope(the change here).